lambeq/rewrite/__init__.py

```diff
@@ -13,7 +13,10 @@
 # limitations under the License.
 
 __all__ = ['RewriteRule', 'CoordinationRewriteRule', 'CurryRewriteRule',
-           'SimpleRewriteRule', 'Rewriter']
+           'SimpleRewriteRule', 'Rewriter', 'DiagramRewriter',
+           'MergeWiresRewriter']
 
 from lambeq.rewrite.base import (RewriteRule, CoordinationRewriteRule,
                                  CurryRewriteRule, SimpleRewriteRule, Rewriter)
+
+from lambeq.rewrite.rewrite_diagram import (DiagramRewriter, MergeWiresRewriter)
```

Comment:
New line.

Reply:
By new line you mean to add another empty line after 21?

Reply:
Yes, all files should end with a newline.